app/Notifications/Mail/EmailChangeNotification.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Notifications\Mail;
+
+use Brewmap\Notifications\MailNotification;
+use Illuminate\Notifications\AnonymousNotifiable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Support\Facades\URL;
+
+/**
+ * @psalm-suppress PropertyNotSetInConstructor
+ */
+class EmailChangeNotification extends MailNotification
+{
+    protected string $userId;
+    protected string $verifyUrl;
+    protected Object $time;
+
+    public function __construct(string $userId)
+    {
+        $this->userId = $userId;
+        $this->time = config("constants.notification.time_to_change_email");
```

Comment:
I think that word "constant" is quite far away of "current time + 90 minutes". You cannot put values like that into configuration - it will be cached on production environment and will fail fo this task.

Reply:
Instead of const, passing `now()->addMinutes(90)` or `90` number will be okay?

Reply:
Something like this? 
```suggestion
        $this->time = Carbon::now()->addMinutes(config("notifications.email_change_timeout"));
```

Reply:
Yep, it looks great! 🙂